templates/gotify/0/rancher-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+catalog:
+  name: "gotify"
+  version: "0.0.1"
+  description: "Service for sending notifications"
+  minimum_rancher_version: v0.46.0
```

Comment:
Where are you getting this minimum version? I can't seem to find anything this might refer to...

chalmers.it | version
-- | --
Rancher | v1.6.14
Cattle | v0.183.37
User Interface | v1.6.37
Rancher CLI | v0.6.7
Rancher Compose | v0.12.5